src/one_dragon/base/notify/push.py

```
@@ -571,26 +571,107 @@ def send_mpnews(self, title, message, media_id, touser="@all"):
+                errmsg = resp.get("errmsg") if resp else None
+                self.log_error(
+                    f"企业微信机器人文字推送失败!status={status}; errcode={errcode}; errmsg={errmsg}; body_snip={body_snip}"
+                )
+
+        # 2. 再发图片
+        if image:
+            image.seek(0)
+            img_bytes = image.getvalue()
+            orig_size = len(img_bytes)
+            if len(img_bytes) <= 2 * 1024 * 1024:
+                # 直接发送，传入图片为png格式，大小≤2MB, 无需压缩
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)